cmd/yggd/main.go

```diff
@@ -25,8 +25,11 @@ import (
 	"google.golang.org/grpc"
 )
 
-var ClientID string = ""
-var ExcludeWorkers map[string]bool = map[string]bool{}
```

Comment:
This patch will need to be updated to support the `--exclude-workers` flag. It was included as part of the original #37 PR, then later added to the yggdrasil-0.2 branch in #62.

Reply:
Updated the code to reflect the ExcludeWorkers